multiqc/modules/skewer/skewer.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse logs from Skewer """
+
+from __future__ import print_function
+
+import os
+from collections import OrderedDict
+import logging
+import re
+from multiqc import config, BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Skewer """
+
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Skewer', anchor='skewer',
+                                            href="https://github.com/relipmoc/skewer",
+                                            info="is an adapter trimming tool specially designed for processing next-generation sequencing (NGS) paired-end sequences.")
+
+        self.sections = list()
+        self.skewer_data = dict()
+        self.skewer_readlen_dist = dict()
+
+        for f in self.find_log_files(config.sp['skewer'], filehandles=True):
+            self.parse_skewer_log(f)
+
+        headers = OrderedDict()
+        headers['pct_trimmed'] = {
+            'title': '% Trimmed',
+            'description': '% of reads trimmed',
+            'scale': 'RdYlGn-rev',
+            'max': 100,
+            'min': 0
+        }
+
+        self.general_stats_addcols(self.skewer_data, headers)
+
+        # Write parsed report data to a file
+        self.write_data_file(self.skewer_data, 'multiqc_skewer')
+        self.write_data_file(self.skewer_readlen_dist, 'multiqc_skewer_readlen_dist')
```

Comment:
Maybe a flag `--write-all-data` so it's user settable? And in the API something like `write_data_file(data, fn, is_extra=False)`? An example could then be

```        
        # Write parsed report data to a file
        self.write_data_file(self.skewer_data, 'multiqc_skewer')
        self.write_data_file(self.skewer_readlen_dist, 'multiqc_skewer_readlen_dist', is_extra=True)
```

(note `, is_extra=True` in the second call)